### PR TITLE
actions updates

### PR DESCRIPTION
**Pull request by kevmoo**
4 commits, Resolve 2 issues, will land seperately

### CODE REVIEW

File: .github/workflows/dart.yml
Comment (jakemac53) [inline .github/workflows/dart.yml:21]: Should we try upstreaming this so it happens by default during setup? (no need to block this on that, but it might be a good change?)

File: mono_repo/lib/src/commands/github/github_yaml.dart
Comment (jakemac53) [inline mono_repo/lib/src/commands/github/github_yaml.dart]: it looks like the `V` type here could be `Map<String, dynamic>`, can't tell for sure

**Review by jakemac53 — APPROVED** [review]

**Comment by jakemac53** [comment]
Although it would be good to address the comments in https://github.com/google/mono_repo.dart/pull/249 before landing or remove that commit here please.

File: .github/workflows/dart.yml
Comment (kevmoo) [reply .github/workflows/dart.yml:21]: Agreed!

I think we'll do this in the actions that @mit-mit is working on.

Until then...

File: mono_repo/lib/src/commands/github/github_yaml.dart
Comment (kevmoo) [reply mono_repo/lib/src/commands/github/github_yaml.dart]: fixed. Rebased. Let's get #249 landed and I'll rebase (again)

**Review by jakemac53 — APPROVED** [review]

**Pull request merged**
